website/static/js/profile.js

```diff
@@ -731,6 +731,11 @@ SocialViewModel.prototype.unserialize = function(data) {
     var self = this;
     var websiteValue = [];
     $.each(data || {}, function(key, value) {
+        if (key === 'profileWebsites') {
```

Comment:
Looks pretty good. One thing, it may not be in the ticket statement, in addition to personal website (profileWebsites), we should also unserialize all other social profile links. See this user, [osf.io/dhyac/](https://osf.io/dhyac/).  @alexschiller what do you think?